src/lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -21,7 +21,7 @@ export default requestIDs => {
     filterArtworksLoader: gravityLoader("filter/artworks"),
     geneArtistsLoader: gravityLoader(id => `gene/${id}/artists`),
     geneFamiliesLoader: gravityLoader("gene_families"),
-    geneLoader: gravityLoader(id => `/gene/${id}`),
+    geneLoader: gravityLoader(id => `gene/${id}`),
```

Comment:
I think you did this in the other PR as well, so would lead to a little merge conflict.

Reply:
Oh wait, nope, this _is_ that other change 😄

Reply:
Yes, this PR is on top of the homepage one!